CONTRIBUTING.md

```diff
@@ -1,56 +1,146 @@
+
+`yarn play` requires the `yarn dev` script to be running in the background.
+
+```sh
+# Start the most recent playground or create one if none exist.
+yarn play
+
+# Start an existing playground or create one if it doesn't exist.
+yarn play my-playground-name
+
+# Force create a new playground
+yarn play --new
+yarn play --new my-playground-name
+
+# Specify a framework (default: next)
+yarn play -f sveltekit
+yarn play --framework sveltekit
 
+# If `-f` is passed with a playground name and the playground
+# already exists, halt. If the playground does not exist, create
+# a new one with that framework.
+yarn play -f sveltekit my-playground-name
+
+# Don't start the project
+yarn play --no-start
+
+# Dry run
+yarn play -n
+yarn play --dry-run
 ```
-cd e2e-projects/next
-yarn slicemachine:dev
+
+## :building_construction: Submit a pull request
+
+> [!NOTE]
+> Code will be reviewed by the Prismic team before merging.[^3]
+>
+> Request a review by opening a pull request.
+
+```sh
+# Open a pull request. This example uses the GitHub CLI.
+gh pr create
+
+# Someone from the Prismic team will review your work. This review will at
+# least consider the PR's general direction, code style, and test coverage.
+
+# When ready, PRs should be merged using the "Squash and merge" option.
 ```
 
-Now you can go to your browser and open `localhost:9999` to see the Slice Machine app up and running with some demo slices and types.
-You're now ready to start working on your changes and see them updated in the app.
+## :rocket: Publish
 
-## E2E tests
+> [!CAUTION]
+> Publishing is restricted to the Prismic team.[^4]
 
-In order to contribute to Slice Machine you will need to maintain and write new e2e tests:
+Use the [**Publish stable**](https://github.com/prismicio/slice-machine/actions/workflows/publish-stable.yml) GitHub Action to publish a new version of Slice Machine and its child packages.
 
-- [Slice Machine e2e tests documentation](./playwright/README.md)
+Version increment types can be specified for each package using the action's workflow form. For example, you can choose to increment `slice-machine-ui` by a major, minor, or patch version. Select "decline" to skip incrementing the version.
 
-<!-- Links -->
+The GitHub Action is powered by a purpose-built script: `script/publish.ts`.
 
-[forum-question]: https://community.prismic.io
-[repo-bug-report]: https://github.com/prismicio/slice-machine/issues/new?assignees=&labels=bug&template=bug_report.md&title=
-[repo-feature-request]: https://github.com/prismicio/slice-machine/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=
-[repo-pull-requests]: https://github.com/prismicio/slice-machine/pulls
+[^1]: This package is maintained by the DevTools team. Prismic employees can ask for help or a review in the [#team-dev-tools](https://prismic-team.slack.com/archives/CPG31MDL1) Slack channel.
```

Comment:
❓ Should this (and the 2 following slack channel links) reference the #dev-tools-dev channel instead?

Reply:
Hum, I'm not even sure we should mention a private internal link, WDYT?

Reply:
Yeah, since this is a public repo, we should remove the links. We could keep the unlinked channel names, though.

Reply:
We decided on slack to keep the links as is 😄